add-domain

```diff
@@ -521,3 +521,4 @@ mailbox_0upgrade.godaddysites.com
 www-rsblox.com
 pdfkinghub.xyz
 randomzee06.s3.au-syd.cloud-object-storage.appdomain.cloud
+data-base.cf
```

Comment:
I can neither confirm or not. the `https://whois.freenom.com/en/whois.html` seems currently down 

drill data-base.cf @9.9.9.9
;; ->>HEADER<<- opcode: QUERY, rcode: NXDOMAIN, id: 43037
;; flags: qr rd ra ; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 0 
;; QUESTION SECTION:
;; data-base.cf.        IN      A

;; ANSWER SECTION:

;; AUTHORITY SECTION:
cf.     5       IN      SOA     a.ns.cf. kohilaire.socatel.cf. 1625949359 10800 3600 604800 5